1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,34 @@
 function readingList(books) {
-  // Write your code here...
+  //   - For each book, create a `<p>` element with the book title and author.
+  for (const book of books) {
+    const pElement = document.createElement("p");
+
+    // we want to assign something inside the p
+    pElement.innerText = `${book.title} by ${book.author}`;
+
+    //   - Add an `<img>` to each book that links to a URL of the book cover.
+    const bookImage = document.createElement("img");
+    bookImage.src = `${book.bookCoverImage}`;
+
+    //   - Use a `<ul>` and `<li>` to display the books and append it to the page.
+    const bookList = document.querySelector("#reading-list");
+    const bookListItem = document.createElement("li");
+
+    bookList.appendChild(bookListItem);
+
+    // Appending the pElement and the image to each <li>
+    bookListItem.appendChild(pElement).appendChild(bookImage);
+
+    //   - Change the style of the book depending on whether you have read it (green) or not (red). Hint: try adding some CSS classes to styles.css for these styles.
+
+    // ***How can we change the colours of each list item?
+    // This works, why does it work - ask Michael!
+    if (book.alreadyRead === false) {
+      bookListItem.classList.add("red");
```

Comment:
Note: This works because `bookListItem` is a reference to the `li` you've created and added to the page. So you can add a class to it here.